README.md

```diff
@@ -1,73 +1,67 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
+# Capstone-Project - Online Conference Website
 
-# Project Name
-
-> Description the project.
+# Description the project.
 
+This project is example of Website for offering services to our clients
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+- html/css/js.
+- following linters standards.
+- following github flows standards.
 
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
+**My project works on mobile and desktop using @Media queries .**
+****************************************************
 To get a local copy up and running follow these simple example steps.
+follow>>>>>>>>>
 
-### Prerequisites
 
 ### Setup
 
-### Install
+to use the project please follow :
 
-### Usage
+ >Open GitHub and go to my GitHub repository name: Capstone-Project-I .to clone.
 
-### Run tests
+ >Click “Code” and copy the given URL.
 
-### Deployment
+ >Open “Git Bash” and change the current working directory to the location where you want the cloned directory.
 
+ >Type git clone in the terminal, paste the URL you copied earlier, and press “enter” to create your local clone.
 
+ >Syntax:
 
-## Authors
+  $ git clone {repository URL}
 
-👤 **Author1**
+### Usage
+this website can be used for commercial reasons or marketing.
 
-- GitHub: [@githubhandle](https://github.com/githubhandle)
-- Twitter: [@twitterhandle](https://twitter.com/twitterhandle)
-- LinkedIn: [LinkedIn](https://linkedin.com/in/linkedinhandle)
+### LIVE DEMO
+CHECK THIS [LINK](https://ogagaoghene.github.io/online-conference-website/)
```

Comment:
- [ ] Please fix the project deployment link because now it's not working.

![image](https://user-images.githubusercontent.com/68796232/175378313-1ddf8fb3-2bba-4c15-bb59-53d0e4ce226b.png)